installer/templates/phx_single/lib/app_name_web.ex

```diff
@@ -80,12 +81,14 @@ defmodule <%= @web_namespace %> do
   end
 
   defp html_helpers do
-    quote do
+    quote do<%= if @gettext do %>
+      # Translation
+      use Gettext, backend: <%= @web_namespace %>.Gettext
+      <% end %>
```

Comment:
This will leave trailing whitespace. I think the `<%= end %>` needs to be at the end of the previous line.